src/gates.jl

```diff
@@ -903,6 +941,48 @@ function CRYGate(θ::Number)
     return round_complex_values(CRY)
 end
 
+@doc raw"""
+    CRYRevGate(θ::Number)
+
+Two-qubit controlled reverse [RYGate](@ref). 
+
+**Circuit Representation**
+```
+     ┌───────┐
+q_1: ┤ RY(ϴ) ├
+     └───┬───┘
+q_0: ────■────
+```
+
+**Matrix Representation**
+
+```math
+\newcommand{\th}{\frac{\theta}{2}}
+
+CRYRev(\theta)\ q_1, q_0 =
+|0\rangle\langle0| \otimes I + |1\rangle\langle1| \otimes RY(\theta) =
+    \begin{pmatrix}
+        1 & 0 & 0 & 0 \\
+        0 & \cos{\th} & 0 & -\sin{\th} \\
+        0 & 0 & 1 & 0 \\
+        0 & \sin{\th} & 0 & \cos{\th}
+    \end{pmatrix}
+```
+"""
+function CRYRevGate(θ::Number)
+    
+    if !(-2*π <= θ <= 2*π)
+        Memento.error(_LOGGER, "θ angle in CRYGate is not within valid bounds")
+    end
+
+    CRY = Array{Complex{Float64},2}([ 1 0 0 0            
```

Comment:
Can you name this CRYRev and on line 983.

Reply:
Fixed